lib/tapioca/dsl/compilers/protobuf.rb

```diff
@@ -78,7 +78,7 @@ class Field < T::Struct
 
         extend T::Sig
 
-        ConstantType = type_member { { fixed: Module } }
+        ConstantType = type_member { { fixed: T::Class[T.anything] } }
```

Comment:
Nice

Reply:
It shouldn't: https://github.com/Shopify/tapioca/blob/main/lib/tapioca/sorbet_ext/backcompat_patches.rb#L14-L24